.github/workflows/set-date.yml

```diff
@@ -38,8 +39,42 @@ jobs:
               }
             }' -f org=$ORGANIZATION -F number=$PROJECT_NUMBER > project_data.json
           echo 'PROJECT_ID='$(jq '.data.organization.projectV2.id' project_data.json) >> $GITHUB_ENV
-          echo 'DATE_FIELD_ID='$(jq '.data.organization.projectV2.fields.nodes[] | select(.name== "End Date") | .id' project_data.json) >> $GITHUB_ENV
-      
+          echo 'DATE_FIELD_ID='$(jq '.data.organization.projectV2.fields.nodes[] | select(.name== "Start Date") | .id' project_data.json) >> $GITHUB_ENV
+
+      - name: Get project data for end date
```

Comment:
Rather than having 2 different steps for the project data, we can also have one single step that extracts the `DATE_FIELD_ID` in two different env variables as `START_DATE_FIELD_ID` and `END_DATE_FIELD_ID`. It's just that, we'll have an unused env variable every time.